pkgs/data/fonts/amiri/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchzip }:
+
+let
+  version = "0.111";
+
+in fetchzip rec {
+  name = "Amiri-${version}";
+
+  url = "https://github.com/alif-type/amiri/releases/download/${version}/${name}.zip";
+
+  postFetch = ''
+    unzip $downloadedFile
+    install -m444 -Dt $out/share/fonts/truetype ${name}/*.ttf
+    install -m444 -Dt $out/share/doc/${name}    ${name}/{*.txt,*.pdf}
+  '';
+
+  sha256 = "1w3a5na4mazspwy8j2hvpjha10sgd287kamm51p49jcr90cvwbdr";
```

Comment:
Could you please put the `sha256` line directly under the `url` line. That increases the readability a bit.

Reply:
Done.